Dockerfile.centos8.dapper

```diff
@@ -1,8 +1,11 @@
 FROM centos:8
 
 # CentOS 8 has reached EOL: https://www.centos.org/centos-linux-eol/
-# Therefore, we need to switch the mirrorlist for the appstream repo to point to http://vault.centos.org
-RUN pushd /etc/yum.repos.d/ && sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* && sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-* && popd
+# Therefore, we need to switch the mirrorlist for the appstream repo
+#
+# point to vault.epel.cloud
```

Comment:
This didn't seem to be necessary for #11 , can we do without it? If not we should push to make the same change in rke2-selinux and k3s-selinux.

Reply:
I think that fixes an intermittent problem, which does not happen as often with vault.epel.cloud.

Again: #12 :-)